lib/auth/auth_with_roles_test.go

```diff
@@ -3118,8 +3118,55 @@ func TestGetAndList_ApplicationServers(t *testing.T) {
 	require.Empty(t, resp.Resources)
 }
 
-// TestListSAMLIdPServiceProviderAndListResources verifies
-// RBAC and search filters when fetching SAML IdP service providers.
```

Comment:
Nit: Should we consider keeping the comment for TestListSAMLIdPServiceProviderAndListResources, or has it become outdated?

Reply:
It is not outdated, but also did not read like that it is explaining all that is happening in detail to me. 

I added it back https://github.com/gravitational/teleport/pull/55744/commits/7be690f514bf4d7df943d540c0bb3f7a343af6d0 seems like its still useful given your question.